jaus_ros_bridge/src/JausDataManager.cpp

```diff
@@ -95,7 +95,7 @@ JausDataManager::JausDataManager(ros::NodeHandle* nodeHandle, udpserver* udp)
   _subscriber_reportRPM = _nodeHandle.subscribe("/thruster_control/report_rpm", 1,
                                                 &JausDataManager::handleReportRPM, this);
   _publisher_setRPM =
-      _nodeHandle.advertise<thruster_control::SetRPM>("/thruster_control/set_rpm", 1, true);
+      _nodeHandle.advertise<thruster_control::SetRPM>("input/jausRosBridge/set_rpm", 1, true);
```

Comment:
@patapufate nit: consider using snake case for topic names, as in:
```suggestion
      _nodeHandle.advertise<thruster_control::SetRPM>("input/jaus_ros_bridge/set_rpm", 1, true);
```

Reply:
fixed in 070444f

Reply:
https://github.com/ChrisScianna/ROS-Underwater-RnD/commit/070444f36525c147f238c8e8078cf2369c16c889 does not update `jaus_ros_bridge` code.

Reply:
Sorry. changed in 1aa9772